hooked/views.py

```diff
@@ -49,8 +49,9 @@ def __init__(self, *args, **kwargs):
     def dispatch(self, request, *args, **kwargs):
         return super(WebhookReceiverView, self).dispatch(request, *args, **kwargs)
 
-    def log_failure(self, message, **kwargs):
-        logger.warning(message, extra={'body': self.request.body}, **kwargs)
+    def log_failure(self, message, extra={}, **kwargs):
```

Comment:
I've updated this with the pattern you suggested @jonprindiville.  Now if you'll excuse me I'm going to find the half dozen places in Lumos I've probably done this previously and correct those as well.